framework/src/problems/FEProblemBase.C

```diff
@@ -3805,6 +3805,14 @@ FEProblemBase::joinAndFinalize(TheWarehouse::Query query, bool isgen)
   }
 }
 
+void
+FEProblemBase::computeUserObjectByName(const std::string & name)
+{
+  TheWarehouse::Query query =
+      theWarehouse().query().condition<AttribSystem>("UserObject").condition<AttribName>(name);
+  computeUserObjectsInternal(EXEC_NONE, Moose::POST_AUX, query);
```

Comment:
`POST_AUX` here seems pretty arbitrary

Reply:
Can you add an argument to this new API that can control the second argument to `computeUserObjectsInternal`?